### ISSUE TITLE
CNN Video not downloading - Unable to download XML

### ISSUE THREAD

**Issue by MaryCoding** [rg3/youtube-dl#4757]
Hi, I am news blogger and was having difficulties trying to download cnn videos. I tried with a link formatted in two different ways and I still cant get a video. Please help me and Great tool! 

1st attempt:
```
youtube-dl -v -o "/media/%(title)s.%(ext)s" http://www.cnn.com/2015/01/20/us/yellowstone-river-spill/index.html
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '-o', '/media/%(title)s.%(ext)s', 'http://www.cnn.com/2015/01/20/us/yellowstone-river-spill/index.html']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.16
[debug] Python version 2.7.6 - Linux-3.13.0-35-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: ffmpeg git-2014-12-27-d4fd3f2, ffprobe git-2014-12-27-d4fd3f2
[debug] Proxy map: {}
[CNNArticle] index.html: Downloading webpage
[CNN] orig-yellowstone-oil-spill: Downloading XML
ERROR: Unable to download XML: HTTP Error 404: Not Found (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 293, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1412, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
```
2nd Attempt:

```
youtube-dl -v -o "/media/%(title)s.%(ext)s" http://www.cnn.com/videos/us/2015/01/20/orig-yellowstone-oil-spill.cnn
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '-o', '/media/%(title)s.%(ext)s', 'http://www.cnn.com/videos/us/2015/01/20/orig-yellowstone-oil-spill.cnn']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.16
[debug] Python version 2.7.6 - Linux-3.13.0-35-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: ffmpeg git-2014-12-27-d4fd3f2, ffprobe git-2014-12-27-d4fd3f2
[debug] Proxy map: {}
[CNNArticle] orig-yellowstone-oil-spill.cnn: Downloading webpage
[CNN] orig-yellowstone-oil-spill: Downloading XML
ERROR: Unable to download XML: HTTP Error 404: Not Found (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 293, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1412, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
```

**Comment by jaimeMF** [comment]
Thanks for the report, it will work in the next version.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.01.22 and newer. See [our FAQ](htps://yt-dl.org/update) if you need help updating.
